### PR TITLE
Add CI for Twig CS Fixer

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.0.x
| Description?      | Remove the old Twig CI and create a dedicated CI to run Twig CS Fixer, as recommended in the Twig documentation: https://twig.symfony.com/doc/3.x/coding_standards.html
| Type?             |  improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | The new CI job named "Twig" will be triggered. If it passes successfully, everything should be ok. UI tests must all be green to ensure no regression has been introduced
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/11593426789
| Fixed issue or discussion?     | -
| Related PRs       | -
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

I found some issues with the Pull Request description:

* The `branch` should be `develop` or `8.2.x`. ([Read explanation](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/pull-requests/#branch))

Would you mind having a look at it? This will help us understand how interesting your contribution is, thank you very much!

(Note: this is an automated message, but answering it will reach a real human)

File: src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Actions/Row/delete_customer_discount.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Actions/Row/delete_customer_discount.html.twig:41]: This one worries me a little, to test if it work as before

File: src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Actions/Row/submit.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Actions/Row/submit.html.twig:52]: Same here

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/preview.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/preview.html.twig:68]: Also needs to be checked if it works as it used to

**Review by jolelievre — DISMISSED** [review]
Thanks @tblivet it is approved for me

I just have a couple of additional comments:
- the license header are all modified in a not very satisfactory way, but it's mainly due to the initial format of the license header in Twig, as we already discussed this header is added and should be validated by our `header-stamp` tool but the CI is not in place yet, so this should be fixed in another PR it's fine for now
- I noticed two or three elements that make me wonder if it will not create a bug like when `'_'` is replaced with `_` without quotes is it correctly replaced, or when `"\n"` is replaced with `'\n'` because of the `\` in simple quotes I worry that the end of line character would not be interpreted correctly Both those worries should be tested manually, probably by a developer

**Comment by tblivet** [comment]
Thanks for the review, @jolelievre!
I checked the Twig templates you mentioned, and it seems to be working as expected, I didn’t notice any issues. 👍

**Review by jolelievre — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
